src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/cover_image.html.twig

```diff
@@ -24,13 +24,13 @@
  *#}
 
 {% block category_cover_image %}
-  {% if coverImage is defined and coverImage is not null %}
+  {% if preview_image is defined and preview_image is not null %}
```

Comment:
Is this twig file still needed with all the refacto you made based on `image_with_preview_widget` ?
Same question for:
- Categories/Blocks/menu_thumbnail_images.html.twig
- Categories/Blocks/thumbnail_image.html.twig

These three files used to be included in Categories/Blocks/form.html.twig but not anymore, if we can remove them all it's for the best!